libs/admin/src/lib/components/admin-page/admin-page.component.html

```diff
@@ -1,21 +1,33 @@
 <mat-progress-bar *rxIf="isLoading$" mode="indeterminate" class="loading-bar" />
-<div class="admin-page">
+<div class="header" *rxLet="currentLang$; let currentLang">
+  <button asset-sg-primary (click)="navigateBack(currentLang)" class="back-button">
+    <svg-icon key="arrow-left" class="close" />
+  </button>
+  @if (isWorkgroupsPage) {
+  <h2>{{ (selectedWorkgroup$ | async)?.name }}</h2>
+  } @else if (isUsersPage) {
+  <h2>{{ (selectedUser$ | async)?.email }}</h2>
+  <asset-sg-chip [type]="'secondary'">{{ "admin.user" | translate }}</asset-sg-chip>
+  } @else {
+  <h2>{{ "menuBar.settings" | translate }}</h2>
```

Comment:
You could move these `translate` pipes into the elements.
Also, string attributes can be passed without `[` brackets `]`.
```suggestion
  <asset-sg-chip type="secondary" translate>admin.user</asset-sg-chip>
  } @else {
  <h2 translate>menuBar.settings</h2>
```

Reply:
Adding the translate pipe directly to the `asset-sg-chip` component does not work. I need to add an additional span, which then does not really shorten anything nor make it more readable IMO